src/libs/EmojiTrie.ts

```diff
@@ -95,9 +96,22 @@ function createTrie(lang: SupportedLanguage = CONST.LOCALES.DEFAULT): Trie<Emoji
     return trie;
 }
 
-const emojiTrie: EmojiTrie = supportedLanguages.reduce((prev, cur) => ({...prev, [cur]: createTrie(cur)}), {});
+// const emojiTrie: EmojiTrie = supportedLanguages.reduce((prev, cur) => ({...prev, [cur]: createTrie(cur)}), {});
+const emojiTrie: EmojiTrie = {
+    en: undefined,
+    es: undefined,
+};
+
+const buildEmojisTrie = (locale: 'en' | 'es') => {
```

Comment:
Based on the above comment we might need to tweak the locale type.

Reply:
```ts
const locales = ['en', 'es'] as const;

type Locale = (typeof locales)[number]; // produces a type of: 'en' | 'es';
```

Reply:
cc @rinej

Reply:
adjusted it, used Locale type, thanks!